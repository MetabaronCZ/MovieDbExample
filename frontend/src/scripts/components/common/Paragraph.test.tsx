import { getByText } from '@testing-library/dom';
import { describe, expect, it } from '@jest/globals';

import { Paragraph } from './Paragraph';
import { TestComponentWrapper, waitForComponent } from 'test-utils/component';

describe('components/common/Paragraph', () => {
  it('should render properly', async () => {
    const { container } = await waitForComponent(
      <Paragraph>CONTENT</Paragraph>,
      { wrapper: TestComponentWrapper },
    );
    expect(container).toMatchSnapshot();

    const content = getByText(container, 'CONTENT');
    expect(content).toBeInTheDocument();

    expect(content.nodeName.toLowerCase()).toEqual('p');
  });
});
