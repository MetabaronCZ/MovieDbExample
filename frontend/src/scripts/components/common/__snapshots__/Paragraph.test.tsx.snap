// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/common/Paragraph should render properly 1`] = `
.c0 {
  font-family: Tahoma,Verdana,Arial,sans-serif;
  font-size: 14px;
  line-height: 24px;
  color: #002d56;
}

<div>
  <p
    class="c0"
  >
    CONTENT
  </p>
</div>
`;
